redis/content.md

```diff
@@ -14,6 +14,12 @@ Redis is an open-source, networked, in-memory, key-value data store with optiona
 $ docker run --name some-redis -d %%IMAGE%%
 ```
 
+Connect to it locally on your machine:
+
+```console
+$ docker run --name some-redis -p 127.0.0.1:6379:6379 -d %%IMAGE%%
+```
```

Comment:
I'd rather not suggest this to users, since this is really something that should be easily inferred by someone familiar with Docker and/or how `-p` works (and we'd prefer to keep the documentation somewhat succinct and specific to Redis instead of a general guide to using Docker).